pre_build

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
-
+#SOURCE_BRANCH=^release/.*$
+SOURCE_BRANCH=^test/.*$
```

Comment:
If you specify this in the script, I highly doubt JB's automation will work since logically it would have to function by setting this variable externally.